runtime/runtime-params-estimator/test-contract/build.sh

```diff
@@ -16,10 +16,33 @@ dd if=/dev/urandom of=./res/medium_payload bs=$(expr 102400 - ${bare_wasm}) coun
 # 1MiB
 dd if=/dev/urandom of=./res/large_payload bs=$(expr 1048576 - ${bare_wasm}) count=1
 
-rm -rf target
 RUSTFLAGS='-C link-arg=-s' cargo +nightly build --target wasm32-unknown-unknown --release  --features small_payload
 cp target/wasm32-unknown-unknown/release/test_contract.wasm ./res/small_contract.wasm
 RUSTFLAGS='-C link-arg=-s' cargo +nightly build --target wasm32-unknown-unknown --release  --features medium_payload
 cp target/wasm32-unknown-unknown/release/test_contract.wasm ./res/medium_contract.wasm
 RUSTFLAGS='-C link-arg=-s' cargo +nightly build --target wasm32-unknown-unknown --release  --features large_payload
 cp target/wasm32-unknown-unknown/release/test_contract.wasm ./res/large_contract.wasm
+
+# Compiling nightly
+
+rm -rf target
+RUSTFLAGS='-C link-arg=-s' cargo +nightly build --target wasm32-unknown-unknown --release --features nightly_protocol_features
+# bare_wasm=$(stat -c%s target/wasm32-unknown-unknown/release/test_contract.wasm)
+bare_wasm=$(stat -f%z target/wasm32-unknown-unknown/release/test_contract.wasm)
+echo ${bare_wasm}
+
+# Generate several files of various sizes. We will compile these files into the Wasm binary to
+# bloat its size to the given values.
+# 10KiB
+dd if=/dev/urandom of=./res/small_payload bs=$(expr 10240 - ${bare_wasm}) count=1
```

Comment:
these *_payload files should be not committed

Reply:
also res/*_contract.wasm

Reply:
ah seems not related to this PR. We should recompile contract with stable now, as near-sdk-rs examples already using `stable-2020-10-08`: 
https://github.com/near/near-sdk-rs/blob/master/rust-toolchain
https://github.com/near/near-sdk-rs/blob/master/examples/fungible-token/build.sh

Reply:
Looks like a bug

Reply:
> also `res/*_contract.wasm`

They are required for sanity check, because we directly embed them into the param estimator, we should probably switch to lazy initialization.